src/testdir/util/check.vim

```diff
@@ -1,5 +1,9 @@
-source shared.vim
-source term_util.vim
+source util/shared.vim
```

Comment:
Does it work, if you use `source ./shared.vim` and `source ./term_util.vim` here?

Reply:
no, because this is sourced from `testdir/` where Vims current directory is `testdir`, so `./term_util.vim` means trying to access `testdir/term_util.vim`.